extensions/package-vulnerability-scanner/src/components/ContentList.vue

```diff
@@ -78,12 +82,73 @@ const tabs = computed<Tab[]>(() => {
 
 const activeTab = ref("All");
 
+const sortOptions = ref<SortOption[]>([
```

Comment:
nit: This one might not need to be a `ref`? It might be good to be just a plain constant or even a frozen entity so it is clear this is the unchangeable catalog of sorting options